README.md

```diff
@@ -57,7 +60,10 @@ Sources:
 
 Warning: `Alt text length should be less than 125 characters`
 
-Alt text should be less than 125 characters in length. The JAWS screen reader reads alt text in 125 character chunks.
+Alt text should be less than 125 characters in length. The JAWS screen reader reads alt text in 125 character chunks. Any information about the image, such as copyright information, image source or extra information should be placed in the caption text below the image.
```

Comment:
@jsnmrs I also added the second sentence from the source, because I think that could be helpful context